config/metadata/note_metadata.yml

```diff
@@ -6,6 +6,9 @@ colophon_tesim: 'Colophon'
 provenance_tesim: 'Provenance'
 note_tesim: 'Note'
 # toc_tesim: 'Table of Contents' (render_table_of_contents_key / value)
+note_admin_tesim: 'AdminNote'
```

Comment:
should it be Admon Note, did you verify the order with Dawn?

Reply:
yes she said it doesn't matter